sdk/identity/Azure.Identity/tests/AzureCliCredentialTests.cs

```diff
@@ -113,5 +113,13 @@ public void AuthenticateWithCliCredential_CanceledByUser()
             AzureCliCredential credential = InstrumentClient(new AzureCliCredential(CredentialPipeline.GetInstance(null), new TestProcessService(testProcess)));
             Assert.CatchAsync<OperationCanceledException>(async () => await credential.GetTokenAsync(new TokenRequestContext(MockScopes.Default), cts.Token));
         }
+
+        [Test]
+        public void ConfigureCliProcessTimeout()
```

Comment:
This and the other tests should also validate that GetToken throws after being cancelled by the (short) timeout just to validate that the option actually applies to the `ProcessRunner`.